wagtail/wagtailadmin/static_src/wagtailadmin/scss/layouts/404.scss

```diff
@@ -0,0 +1,101 @@
+@import 'wagtailadmin/scss/variables';
+@import 'wagtailadmin/scss/mixins';
+@import 'wagtailadmin/scss/grid';
+@import 'wagtailadmin/scss/components/logo';
+
+body {
+    font-family: $font-sans;
+    color: $color-white;
+
+    .page404__bg {
+        position: fixed;
+        top: 0;
+        left: 0;
+        width: 100%;
+        height: 100%;
```

Comment:
No need for fixed positioning here. Just make the page take up the whole space by setting its height to `100vh`.